Makefile

```diff
@@ -186,14 +186,18 @@ stylesheets: public/css/index.css
 
 .IGNORE: public/css/index.css
 public/css/index.css: $(STYLESHEETS)
-	lessc $< $@
+	@hash lessc > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		docker run --rm -v $(PWD):/app appleboy/node-less $< $@; \
+	else \
+		lessc $< $@; \
+	fi
```

Comment:
Okay, this is good for now. If we do a full node build for assets, I think adding it as a hard dependency might be best, it's usually just a `apt-get` away, no environment setup necessary.